distribution/test/itests/test-itests-catalog/src/test/resources/metacard4.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<!-- /**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/ -->
+<metacard xmlns="urn:catalog:metacard" xmlns:ns2="http://www.opengis.net/gml"
+          xmlns:ns3="http://www.w3.org/1999/xlink" xmlns:ns4="http://www.w3.org/2001/SMIL20/"
+          xmlns:ns5="http://www.w3.org/2001/SMIL20/Language">
+    <type>ddf.metacard</type>
+    <source>ddf.distribution</source>
+    <string name="metadata-content-type-version">
+        <value>myVersion</value>
+    </string>
+    <string name="title">
+        <value>Metacard-4</value>
+    </string>
+    <geometry name="location">
+        <value>
+            <ns2:Polygon>
+                <ns2:exterior>
+                    <ns2:LinearRing>
+                        <ns2:pos>-112.267735 33.469045</ns2:pos>
+                        <ns2:pos>-112.269967 33.468007</ns2:pos>
+                        <ns2:pos>-112.269430 33.466396</ns2:pos>
+                        <ns2:pos>-112.265761 33.467344</ns2:pos>
+                        <ns2:pos>-112.267735 33.469045</ns2:pos>
+                    </ns2:LinearRing>
+                </ns2:exterior>
+            </ns2:Polygon>
+        </value>
+    </geometry>
+    <dateTime name="created">
+        <value>2015-04-18T10:50:27.371-07:00</value>
+    </dateTime>
+    <dateTime name="modified">
+        <value>2015-04-18T10:50:27.371-07:00</value>
+    </dateTime>
+    <base64Binary name="thumbnail">
+        <value>CA==</value>
+    </base64Binary>
+    <string name="metadata">
+        <value>&lt;?xml version="1.0" encoding="UTF-8" standalone="yes"?&gt;
+            &lt;Resource&gt;
+            &lt;name&gt;Connexta&lt;/name&gt;
```

Comment:
Would you like me to change the geometry as well? It's a polygon around the office.